### ISSUE TITLE
This downloader only downloads png files

### ISSUE THREAD

**Issue by JohnChu101** [OtakoidTony/KakaoEmoticonDownloader#1]
Is it possible to download animated files? According to my analysis, the animated emoticon are actually webp files, but i can't read them on chrome or any image viewer. The signature inside the file says "FROM Kakao.Corp. WebpAnimator", the content might be encrypted but i don't know how to decrypt.

**Comment by rojikku-dev** [comment]
No, this can't download animated files. Because this just downloads Sample Images.
